DevelopedCalculator.js

```diff
@@ -9,9 +10,13 @@ class DevelopedCalculator extends Calculator {
         this.hostElem.prepend(historyDiv);
 
 
-        this.updateResultElement();
+        const currentResultUpdate = []
+    }
 
+    onButtonNumberClick_(event) {
+        super.history.push(this.currentResult)
```

Comment:
Where `super.history` is defined?

Reply:
I was not know super needs defining 

because you did not told me to define in when I added : line 4
  super.render()

Reply:
`super` refers to the parent class 
is `history` defined in the super class ?

Reply:
I can not see any class has super name

Reply:
I can not see any class has super name

Reply:
`super` is keyword, refers to the parent class 

your class `extends` `Calculator` class 
`super` keyword allow you access properties and method on the parent class 

`super.render()`
you called the `render` method of `Calculator` class

Reply:
Thanks a lot for clearing

Reply:
any update on this?